### DISCUSSION TITLE
12V i -8V

### DISCUSSION THREAD

**Discussion by stenet2** [ClassicDIY/SkyeTracker#50] [General]
Is it normal that when the actuator is extended there is 12V and when the actuator is retracted it is -8V?

**Comment by graham22** [comment]
No, there might be an issue with your controller board or you can
measure the current to the actuator, if it's higher while retracting then it could be a faulty actuator

**Reply by stenet2** [reply]
I've already changed ESP, engine board, firmware and it's always the same

**Comment by graham22** [comment]
I would try supplying 12V directly to the actuator while measuring the current for each direction (polarity).
if the current is the same, then test the controller board without the esp by providing a 3.3V signal to the control pins for both directions...

**Comment by stenet2** [comment]
Ok, the problem was the drivers. The L6384E system, which controls the transistors, does not fully control one of them, and as a result one transistor was heated and did not provide full voltage to the motor. I changed to the BTS7960B controller and it's ok. Thanks for your help

**Comment by graham22** [comment]
Glad you found the problem...

On Fri, Feb 16, 2024 at 12:18 PM stenet2 ***@***.***> wrote:

> Ok, the problem was the drivers. The L6384E system, which controls the
> transistors, does not fully control one of them, and as a result one
> transistor was heated and did not provide full voltage to the motor. I
> changed to the BTS7960B controller and it's ok. Thanks for your help
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClassicDIY/SkyeTracker/discussions/50#discussioncomment-8495199>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIYATOGGSUXEJYKHJJYKA3YT6IEVAVCNFSM6AAAAABDADCPVWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIOJVGE4TS>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Graham A. Ross
(613) 769-1331
